Sources/XMLCoder/Decoder/XMLKeyedDecodingContainer.swift

```diff
@@ -292,22 +299,27 @@ extension XMLKeyedDecodingContainer {
             return ((cdata as? StringBox)?.unboxed as? T) ?? emptyString
         }
 
-        switch strategy(key) {
-        case .attribute?:
-            box = try getAttributeBox(for: type, attributes, key)
-        case .element?:
-            box = try getElementBox(for: type, elements, key)
-        case .elementOrAttribute?:
-            box = try getAttributeOrElementBox(attributes, elements, key)
-        default:
-            switch type {
-            case is XMLAttributeProtocol.Type:
+        let box: Box
+        if key.stringValue != "" {
```

Comment:
We can use `!isInlined` here with the computed property suggestion

Reply:
done

Reply:
Thanks for the quick change! Will review in a bit